README.md

```diff
@@ -130,29 +152,41 @@ Also, we collect here other resources that compare different tools:
 
 * **Short description**: Interactive theorem prover
 * **URL**: https://coq.inria.fr/
-* **License**:
-* **Modeling language**: dependently-typed lambda calculus
-* **Domain**: general
-* **Abstraction level of models**: any
-* **Supported languages**:
+* **License**: LGPL 2.1
+* **Modeling language**: Dependently-typed lambda calculus.
+* **Domain**: General.
+* **Abstraction level of models**: Any.
+* **Supported languages**: OCaml, Haskell, Scheme.
 * **Analysis mechanisms & automation**: 
-    * interactive theorem proving with user-definable tactics (Ltac language to define tactics)
-    * QuickCheck for finding counterexamples
+    * Interactive theorem proving with user-definable tactics (Ltac language to define tactics).
+    * QuickChick for finding counterexamples.
 * **UX (UI, tooling, libraries)**:
     * Built-in IDE, and also integrations for VSCode, Emacs, and Vim
-* **Scalability**:
-* **Stability**:
+* **Scalability**: Has an excellent module system, so scales well through modularity.
```

Comment:
We also wanted to describe whether the analysis mechanisms can scale when adding hardware (cores or multiple machines); IIRC, Coq had some support for parallel proof processing, is that correct?

Reply:
It has poor support. There is a Coq-IDE support for resolving goals in parallel, but running the `coqc` compiler does not do this. It's mainly a single core environment at the moment.

Reply:
Thanks! I'll add that as a note then.